metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -313,6 +314,19 @@ object UserConfiguration {
           |launcher, not available in PATH.
           |""".stripMargin,
       ),
+      UserConfigurationOption(
+        "projects-roots",
```

Comment:
I wonder if need the setting if this is done automatically. Seems a bit complex and people might never use workspaces, which makes this setting their roots more difficult

Reply:
As above: automatically it's only for shallow levels. Yeah... This setting is a bit far from perfect. But for most users it won't be needed.

As I mentioned in the feature request, we should probably switch to [workspace/configuration](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_configuration) (from didChangeConfiguration) in the long run. It allows for scoping settings, so making settings per workspace folder will be much more intuitive.

Reply:
Per workspace would be ideal here, we can introduce the setting with that change then, maybe? OTherwise we will be stuck with this much more complex setting

Reply:
I deleted the setting for now and left only auto detection.

Reply:
Could you remove it from UserConfiguration for now also?